application/app/mvnw

```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```

Comment:
I guess we could remove the whole Maven wrapper as we run the build from within a container.

Reply:
Ah, okay. Will do.